src/olympia/api/fields.py

```diff
@@ -387,11 +362,19 @@ def to_representation(self, value):
         return {'url': data, 'outgoing': outgoing}
 
 
+class URLTranslationField(serializers.URLField, TranslationSerializerField):
+    pass
+
+
+class EmailTranslationField(serializers.EmailField, TranslationSerializerField):
+    pass
+
+
 class OutgoingURLField(OutgoingSerializerMixin, serializers.URLField):
     pass
 
 
-class OutgoingTranslationField(OutgoingSerializerMixin, TranslationSerializerField):
+class OutgoingURLTranslationField(OutgoingSerializerMixin, URLTranslationField):
```

Comment:
Can you rename the ES variant of this field accordingly ?

Reply:
The ES variant technically doesn't inherit from a `URLTranslationField`/`URLField` so I didn't change it (doesn't need to - the only difference between a `CharField` and a `URLField` is the validation, which isn't used for read-only fields) but I can for naming consistency, yep.